libraries/libfc/secp256k1/CMakeLists.txt

```diff
@@ -5,8 +5,7 @@ add_library(secp256k1-internal INTERFACE)
 target_include_directories(secp256k1-internal INTERFACE secp256k1/src)
 
 target_compile_definitions(secp256k1-internal INTERFACE ENABLE_MODULE_RECOVERY=1
-                                                        ENABLE_MODULE_EXTRAKEYS=1
-                                                        COMB_BLOCKS=11
+                                                        COMB_BLOCKS=43
```

Comment:
These `COMB` settings come from
https://github.com/bitcoin-core/secp256k1/blob/3fdf146bad042a17f6b2f490ef8bd9d8e774cdbd/CMakeLists.txt#L93-L105
notice the default setting is now 86 and the `COMB` defines match what is here.